docs/fundamentals/runtime-libraries/snippets/System/Double/CompareTo/csharp/compareto2.cs

```diff
@@ -7,10 +7,8 @@ public static void Main()
     {
         double value1 = 6.185;
         double value2 = value1 * .1 / .1;
-        Console.WriteLine("Comparing {0} and {1}: {2}\n",
-                          value1, value2, value1.CompareTo(value2));
-        Console.WriteLine("Comparing {0:R} and {1:R}: {2}",
-                          value1, value2, value1.CompareTo(value2));
+        Console.WriteLine($"Comparing {value1} and {value2}: {value1.CompareTo(value2)}\n");
```

Comment:
same xplat suggestion:

```suggestion
        Console.WriteLine($"Comparing {value1} and {value2}: {value1.CompareTo(value2)}{Environment.NewLine}");
```